### ISSUE TITLE
Facebook for woocommerce not working

### ISSUE THREAD

**Issue by rosesnpurls** [facebookincubator/facebook-for-woocommerce#1004]
I've deactivated and reactivated, uninstalled and reinstalled, tried going in through FB business manager - nothing is working

 "Your API key is no longer valid. Please click "Settings > Advanced Options > Update Token"." - Go to settings, get "sorry something went wrong"
Your Facebook page URL https://www.facebook.com/rosesnpurls/
Your website URL www.rosesnpurls.com
Current version of Facebook-for-WooCommerce 1.9.12, WooCommerce 5.2, WordPress 4.9.1

**Comment by deargeek** [comment]
Yes, same problem here, with:

Facebook for WooCommerce 1.9.10
WooCommerce 3.6.3
WordPress 5.2

**Comment by splozm** [comment]
I have been trying to connect - full stop. I cannot add sync my store using the woocommerce plugin. I follow the plugin wizard to the facebook page step and get the following message

" A Facebook Page can only be connected to one store at a time. Clare's Toys is already in use by another WooCommerce store. You can remove the existing connection to use the Page for this store, or you can select a different Page."

I have installed/reinstalled/deleted/cleaned database
I have tried deleting the shop on Facebook but it still tells me there is a connection.

I cannot remove existing connection manually or in the wordpress woocommerce plugin wizard by clicking on the remove exisitng connection button. When i try it says 

"Sorry, something went wrong. This might be a temporary issue and we're working to get it fixed as soon as we can. Please try again later."

This has been going on for several days now.

Facebook-for-WooCommerce 1.9.12, WooCommerce 5.2, WordPress 4.9.1 

I had to install the pixel through GTM just to get some data.

If you fix it i will be able to advertise my products on Instagram and you will get money for that.

**Comment by Avakiano** [comment]
hello,

Its important that you sign in on facebook with the correct account, i had that issue, because it automatically sign in the last account used on fb. so check it out.

@splozm 

> I have installed/reinstalled/deleted/cleaned database
> I have tried deleting the shop on Facebook but it still tells me there is a connection.

after you "manage settings" in the plugin, you must delete all the config in the "advanced setting"

but even when it connects, and everything its ok, it doesn't appear the labels on IG.

regards!

**Comment by hribare** [comment]
Same problem here
WC Version: 3.5.2

**Comment by jacobabey1** [comment]
Report it from  this form: https://www.facebook.com/help/contact/268228883256323

**Comment by sanchezpu** [comment]
Hello, has anyone found the solution ?, I have the same problem. I spoke with Facebook and they sent me to woocomerce.com and from woocomerce.com they sent me here

**Comment by jacobabey1** [comment]
https://developers.facebook.com/community/threads/1066316133570771/

**Comment by nguyenanh17** [comment]
same issue here

**Comment by yinghec** [comment]
## Automated Support Message

Hello,

Thanks for your interest in our plugin! Our developer team tries to provide personal support on GitHub whenever possible.

If you have a feature request or feedback for the developers, reply with `Feature Request` or `Feedback`.
If you are a developer or have a question for the developers related to code, reply with `Developer Question`.
If you are running ads with us, reply with `Advertiser`.

If you have encountered a bug or a problem with the plugin, keep reading for information that can help you resolve the issue yourself or provide us details we need to help you.

## Search Existing Issues

First, please [search our issues](https://github.com/facebookincubator/facebook-for-woocommerce/issues?q=) to see if your problem has been reported before.
There may already be a workaround or a solution to your problem. You can use the [search bar](https://github.com/facebookincubator/facebook-for-woocommerce/issues?q=) near the top of our issues page. Be sure to search both _open_ and _closed_ issues.
If you do find a similar issue, leave a comment on that issue instead of creating a new one. Your comment helps us prioritize which existing issues are important and urgent.

## Common Resolution Methods

If you are experiencing a problem or bug and have not found any similar issue, please try the following:

* Upgrade to the latest version of the plugin
* Try to `Resync` products using the button on the plugin config page
* Reset the plugin by selecting `Manage Settings > Advanced Options > Delete` from the configuration page of the plugin and setting up again
* Try deactivating other plugins that you think may be interfering, then reset and set up again

## Necessary Information

If the above steps did not help, please provide the following information in your next comment in order for us to debug your problem:

* WooCommerce version
* WordPress version
* Plugin version (please note: we only support up-to-date versions of the plugin)
* A screenshot of where you see the problem (for example: if it is on Facebook, provide a screenshot of Facebook, if it is on WooCommerce, provide a screenshot of WooCommerce. You can provide multiple screenshots if the problem relates to different screens.)
* The URL of your website
* The URL of your Facebook Page
* The URL(s) of where you took your screenshot(s)
* If applicable, detailed steps you took in order to see the problem

Please include `Debug Ready` in your reply so our automated system can note your issue for manual review. Please be aware that due to a high volume of issues, we may not be able to respond immediately.

## Response Time

We prioritize issues with the tags `Advertiser` or `Multiple Reports`.
For issues that have the tags `Debug Ready`, `Feature Request` or `Developer Question`, we aim to respond within 30 days.

If you have not responded to our automated message with one of the above phrases, such as `Debug Ready`, we may close your issue after 30 days. If a `Feature Request` is inactive (without new requests, messages or pull requests) for 6 months, we may close it.

Please note that we are not always able to provide support for issues related to interaction from other plugins with this plugin or related to the Facebook advertising interface. To get support related to Facebook ads, try visiting the [Facebook Ads Help Center](https://www.facebook.com/business/help).

For more information on our response times, visit [Facebook Support on GitHub](https://github.com/facebookincubator/facebook-for-woocommerce/wiki/Facebook-Support-on-GitHub).

Thank you,
The Facebook Dev Team

**Comment by Luther2010** [comment]
## Automated Support Message - Issue Closed

Hello,

Because it has been 30 days since this issue was updated and you have not included `Debug Ready` in your reply, our automated system will now close your issue.
If you are still having this problem, please open a new issue and reference this one.

For more information on our response times and support policy, visit [Facebook Support on GitHub](https://github.com/facebookincubator/facebook-for-woocommerce/wiki/Facebook-Support-on-GitHub).

Thank you,
The Facebook Dev Team

**Comment by PProtonzz** [comment]
Sorry for late reply.

I think we can solve this issue by 3 type.

1. Use latest Facebook for Woo-commerce plugin.

2. Don't use new Facebook account, wait at-least 1 day.

3. Below.

I often see this issue come up when there are non-ASCII characters in the page title. I took a look and noted that there are some non-ASCII characters in the title of the site. This data gets saved to Facebook in a field called wc_facebook_external_business_id and we will want to change this to see if it allows your connection to Facebook to complete successfully.

There is a method to change this within the site itself, so let’s start by trying out the steps to change this to see if it helps:

Go to https://pranavfulkari.com/ wp-admin/options.php
Search for wc_facebook_external_business_id
In the field next to it, change the data in that field to just the basic title of the page without any extra characters in place.
Scroll to the bottom of this page and click Save Changes.
In some cases, the data may not be saved due to larger databases and / or PHP settings, so after saving this, locate the wc_facebook_external_business_id field again and verify that the changes in place did save.
Once this is done, can you please try to connect to Facebook again and let me know if you get any further in the connection process?

Cheers,
Pranav Fulkari

**Comment by rosesnpurls** [comment]
Nome of those are applicable, I have no non-ASCII characters in my domain. I switched platforms because they plugin didn’t work; I’m now using Square.

-Jen @ Roses & Purls
